components/hab/src/cli.rs

```diff
@@ -917,10 +916,9 @@ pub fn sub_sup_run() -> App<'static, 'static> {
                   GCrBOW6CCN75LMl0j2V5QqQ6nNzWm6and9hkKBSUFPI=')")
     (@arg CHANNEL: --channel +takes_value
         "Receive Supervisor updates from the specified release channel [default: stable]")
-    (@arg BLDR_URL: -u --url +takes_value {valid_url}
+    (@arg BLDR_URL: -u --url default_value("https://bldr.habitat.sh") {valid_url}
```

Comment:
These are good improvements, but if we're making them to an argument that gets used many places we should update *all* the instances at once.

Similarly for `--group`, `--strategy`, `--binding-mode` and `--health-check-interval`

Reply:
Wow, you're right, we do use this option in a lot of spots. I foresee more work in testing and refactoring the cli bits to not use the macro. I would prefer to keep this one as focused as I can and tackle the other occurrences as I come to them. My hope is that using the builder style syntax will allow us to reuse more code and maybe eliminate a lot of the copy/pase stuff necessary when using the macro.

Reply:
If you want to make these improvements in a later PR, that's fine. But if that's the case, don't improve *any* of them. I think inconsistency is worse because when people later search to try to fix things up, it'll be harder to find them all because they'll look different.

Reply:
So after this PR merges the next thing is to refactor this function. I agree that inconsistency is a bad thing, but I don't think it's unreasonable to take refactors in small bits.

Reply:
Nothing wrong with small pieces, but the nature of those pieces is important since work often gets stopped not exactly where we were planning it to. Every change should be conceptually atomic. So, it's fine to make that unit "fix all of `hab sup run`" or "fix all the `--url` args", but when there's an intersection like this, I don't like the resulting situation of introducing inconsistency among pieces of code that that are semantically identical. Breaking it down like:

1. Fix all the `hab sup run`-specific bits
1. Fix all the `--url` args
1. Fix all the `--group` args
1. etc…

Seems cleaner to me